Azure/communication-ui-library-android
# [Bug] Switch pip camera button accessibility

## Purpose

* Announce the front/ back camera when user tap on the pip camera image button

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test


- Turn on the Talkback
- Navigate to call setup page.
- Navigate to Join call and activate it.
- Turn Video ON
- Navigate to PIP of local video.
- Get focus on 'Switch camera button'  and activate it.
- Observe the issue.


## What to Check
Verify that the following are valid
* Talkback announce the front/ back camera

## Other Information
